tdrs-backend/tdpservice/test/test_admin.py

```diff
@@ -0,0 +1,47 @@
+"""System wide tests.
+
+Fixtures are used to create a regular user
+and an admin user. The regular user has is_staff and is_superuser
+set to false, while the admin_user has both set to true.
+"""
+
+import pytest
+from django.urls import reverse
+from rest_framework import status
+
+
+@pytest.mark.django_db
+def test_can_access_admin(client, admin_user):
+    """Test an authenticated admin_user can access the admin console."""
+    client.login(username=admin_user.username, password='test_password')
+    url = reverse('admin:index')
+    response = client.get(url)
+    assert response.status_code == status.HTTP_200_OK
+    assert "Django administration" in response.rendered_content
```

Comment:
```suggestion
    from django.contrib import admin
    assert admin.site.site_header in response.rendered_content
```
The site header is a modifiable property of the admin site, so changes to it could cause this test to fail when a hard-coded string is used.

Reply:
This has been updated to include all elements that should be there